shared/constants/chat2/index.js

```diff
@@ -31,7 +31,7 @@ export const makeState: I.RecordFactory<Types._State> = I.Record({
     [pendingConversationIDKey, makeConversationMeta({conversationIDKey: noConversationIDKey})],
   ]),
   moreToLoadMap: I.Map(),
-  lastReadMessageMap: I.Map(),
+  orangeLineMap: I.Map(),
```

Comment:
Renamed `lastReadMessageMap` back to `orangeLineMap` to make it clearer that this map should really only be used to display an orange line.